### ISSUE TITLE
KL loss of Variational  Autoencoder

### ISSUE THREAD

**Issue by CicoZhang** [rasbt/deep-learning-book#59]
I did not find the KL loss in this example, https://github.com/rasbt/deep-learning-book/blob/master/code/model_zoo/pytorch_ipynb/autoencoder-var.ipynb, Is only this cost `cost_fn = torch.nn.BCELoss()` used?

**Comment by rasbt** [comment]
Thanks for the note! I remember adding that ... hm ...

I see that the first version that I implemented included the KL div component, indeed (https://github.com/rasbt/deep-learning-book/blob/1d3ab598b9184491b5e9a4107fd4e249baced540/code/model_zoo/pytorch_ipynb/autoencoder_var.ipynb). I must have accidentally removed it when I "cleaned up" the implementations and copy&pasted the training procedure from a regular autoencoder. Anyways. will add it back
